### ISSUE TITLE
evennia stop + ctrl-C in windows causes traceback

### ISSUE THREAD

**Issue by TehomCD** [evennia/evennia#1125]
#### Brief summary of issue / Description of requested feature:
The latest commit causes a traceback to be generated upon using evennia stop and hitting ctrl-C to bypass the repeated 'stop batch job (y/n)' prompt:
` Traceback (most recent call last):
   File "C:\arx\Lib\site-packages\twisted\application\app.py", line 395, in startReactor
     self.config, oldstdout, oldstderr, self.profiler, reactor)
   File "C:\arx\Lib\site-packages\twisted\application\app.py", line 310, in runReactorWithLogging
     reactor.run()
   File "C:\arx\Lib\site-packages\twisted\internet\base.py", line 1195, in run
     self.mainLoop()
   File "C:\arx\Lib\site-packages\twisted\internet\base.py", line 1204, in mainLoop
     self.runUntilCurrent()
 --- <exception caught here> ---
   File "C:\arx\Lib\site-packages\twisted\internet\base.py", line 798, in runUntilCurrent
     f(*a, **kw)
   File "C:\arx\Lib\site-packages\twisted\internet\base.py", line 581, in stop
     "Can't stop reactor that isn't running.")
 twisted.internet.error.ReactorNotRunning: Can't stop reactor that isn't running.` It's essentially just a display error, as far as I can tell, but hitting ctrl-C will continue to generate it each time, which can be confusing.

#### Steps to reproduce the issue / Reasons for adding feature:

1. evennia stop
2. ctrl-c
3. ctrl-c

#### Error output / Expected result of feature
The above traceback, printed each time ctrl-c is entered.

#### Extra information, such as Evennia revision/repo/branch, operating system and ideas for how to solve / implement:
I'll take a look and see if I can figure it out tonight, and write a pull request if so. Probably a lot easier for me to do so since I use my windows machine for testing stuff.

**Comment by TehomCD** [comment]
I poked around this for a while, but no matter where I tried to catch ReactorNotRunning in evennia_launcher, it's just consistently being raised when Evennia stops on windows. At least it's just a small display error and doesn't seem to impact anything.

**Comment by Griatch** [comment]
I saw this error during testing for #1100 but not if running on a new Console with a previously fully stopped and started Evennia. I have not seen this Ctrl-C based traceback since. Worth checking.

Windows is *extremely* finicky about sending signals though so in the worst case this is something time-dependent or even Windows dependent (I tested it on Win7). 

The Windows way of sending a SIGINT-like signal (`win32api.GenerateConsoleCtrlEvent(0, 0)` will go to *every* process in the console, meaning both `evennia_launcher` (which you use to send `evennia stop`) and `evennia_runner` (which holds the loop that runs/restarts the Server/Portal) must handle `KeyboardInterrupt` exceptions in the same way that the Server/Portal themselves must. I do this with `try ... except KeyboardInterrupt` check. The error can also be handled by turning off Ctrl-C handling using `win32api.SetCtrlConsoleHandler(None, True)` in the main loop of `evennia/server/evennia_runner.py` (use `False` to turn it back on). When doing this I had an issue with appearing to not actually stop the Server though, but get a "port-still-used" error when starting the server later. Possibly one could turn/off the Ctrl handling somehow in the runner and server to
